packages/owl-bot/cloud-build/update-pr.yaml

```diff
@@ -13,6 +13,16 @@
 # limitations under the License.
 
 steps:
+  # Populate credentials for cloning from private repo
+  - name: 'bash'
+    args:
+      - '-eEuo'
+      - 'pipefail'
+      - '-c'
+      - |-
+        echo https://x-access-token:$$GITHUB_TOKEN@github.com >> $${HOME}/.git-credentials
+    env:
+      - 'GITHUB_TOKEN=$_GITHUB_TOKEN'
```

Comment:
Can we now remove lines 51 through 60?  Have you tried running this build manually?

Reply:
Actually no. Is there a guide for running it locally?

Keeping the git credentials in the git clone is a safe guard for being non disruptive (it will continue to work even if the first step is doing it wrong).

I'm happy to remove if it's not necessary.